content/documentation/staging/tutorial/03-first-steps/03-open-application-to-outside.adoc

```diff
@@ -0,0 +1,97 @@
+
+icon:bullhorn[size=1x]{nbsp} There are multiple ways to create a DNS entry depending of the platform, servers or services that you are using.
+
+For minikube we will check the External IP of the Ingress gateway:
+
+[source,bash]
+----
+$ kubectl get services/istio-ingressgateway -n istio-system
+NAME                   TYPE           CLUSTER-IP     EXTERNAL-IP    PORT(S)                                                                      AGE
+istio-ingressgateway   LoadBalancer   10.101.6.144   10.101.6.144   15021:30757/TCP,80:32647/TCP,443:30900/TCP,31400:30427/TCP,15443:31072/TCP   19h
+----
+
+And we will create a simple entry in the `/etc/hosts` of the tutorial machine with the desired DNS entry:
+
+[source,bash]
+----
+...
+10.101.6.144 control.travel-control.istio-cluster.org
+...
+----
+
+Then from this machine, the url `control.travel-control.istio-cluster.org` will resolve to the External IP of the Ingress Gateway of Istio.
+
+NOTE: *[2]* Use the Request Routing Wizard on *control* service to generate a traffic rule
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/03-03-service-actions.png" title="Request Routing Wizard">
+    <img src="/images/tutorial/03-03-service-actions.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
+Any request will be routed to the *control* workload.
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/03-03-request-routing.png" title="Routing Rule">
+    <img src="/images/tutorial/03-03-request-routing.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
+Use the Advanced Options and add a gateway with host `control.travel-control.istio-cluster.org` and create the Istio config.
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/03-03-create-gateway.png" title="Create Gateway">
+    <img src="/images/tutorial/03-03-create-gateway.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
+Verify the Istio configuration generated.
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/03-03-istio-config.png" title="Istio Config">
+    <img src="/images/tutorial/03-03-istio-config.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+NOTE: *[3]* Test the *control* service
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/03-03-test-gateway.png" title="Test Gateway">
+    <img src="/images/tutorial/03-03-test-gateway.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+NOTE: *[4]* Review *travel-control* namespace in Kiali
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/03-03-travel-control-graph.png" title="Travel Control Graph">
+    <img src="/images/tutorial/03-03-travel-control-graph.png" style="display:block;margin: 0 auto;" />
```

Comment:
The image is not 100% accurate as it shows destination services & workloads as if they were in the mesh, but they aren't yet at this point of the tutorial
Here's what I have at this step: 
![Capture d’écran de 2021-01-18 14-14-07](https://user-images.githubusercontent.com/2153442/104920080-8e768d80-5997-11eb-9d52-6955329b6db0.png)

It's not such a big problem, but perhaps is a bit misleading for the users.

Reply:
hmm perhaps you're right there's something wrong on my side. I'll double check

Reply:
I agree with this, at this point in the tutorial we have only instrumented the control workload.  So we should not have telemetry out of control.  Actually, for me I don't even have the requests to the travel-portal services.

Reply:
I've repeat the very same step several times in minikube platform, and now with Kiali v1.29.0 I have a similar image:
![image](https://user-images.githubusercontent.com/1662329/105716474-25ef5980-5f1f-11eb-875f-52051a9fb213.png)

So, I'm curious if the differences may be due timeline (did you let the travel control app creates enough traffic ?) or platform specific.

Reply:
I don't think it was a matter of insufficient traffic, I did let it run for minutes, perhaps tens of minutes.
I'm going to retry it perhaps tomorrow. But this isn't a blocker anyway. If we find something to add to the tutorial, we can still add it later.
As of now, the tutorial is in a very good shape!